Matplotlib created a temporary config/cache directory at /tmp/matplotlib-8wf9slnt because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-o_o73koe because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Matplotlib created a temporary config/cache directory at /tmp/matplotlib-g3ctp_hk because the default path (/home/mriqc/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Traceback (most recent call last):
  File "/opt/conda/bin/mriqc", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.9/site-packages/mriqc/cli/run.py", line 167, in main
    mriqc_wf.run(**_plugin)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/workflows.py", line 638, in run
    runner.run(execgraph, updatehash=updatehash, config=self.config)
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 184, in run
    self._clean_queue(jobid, graph, result=result)
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 256, in _clean_queue
    raise RuntimeError("".join(result["traceback"]))
RuntimeError: Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/mriqc/engine/plugin.py", line 60, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 527, in run
    result = self._run_interface(execute=True)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 645, in _run_interface
    return self._run_command(execute)
  File "/opt/conda/lib/python3.9/site-packages/nipype/pipeline/engine/nodes.py", line 771, in _run_command
    raise NodeExecutionError(msg)
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node synthstrip.

Cmdline:
	synthstrip -b 1 -i /scratch1/03201/jbwexler/work_dir/mriqc/ds004056_sub-22/mriqc_wf/anatMRIQC/synthstrip_wf/_in_file_..scratch1..03201..jbwexler..openneuro_derivatives..derivatives..mriqc..ds004056-mriqc..sourcedata..raw..sub-22..ses-01..anat..sub-22_ses-01_acq-MP2R_T1w.nii.gz/pre_n4/clipped_corrected.nii.gz --model /opt/freesurfer/models/synthstrip.1.pt -o clipped_corrected_desc-brain.nii.gz -m clipped_corrected_desc-brain_mask.nii.gz
Stdout:
	Configuring model on the CPU
	Using custom model weights
	Input image read from: /scratch1/03201/jbwexler/work_dir/mriqc/ds004056_sub-22/mriqc_wf/anatMRIQC/synthstrip_wf/_in_file_..scratch1..03201..jbwexler..openneuro_derivatives..derivatives..mriqc..ds004056-mriqc..sourcedata..raw..sub-22..ses-01..anat..sub-22_ses-01_acq-MP2R_T1w.nii.gz/pre_n4/clipped_corrected.nii.gz
Stderr:
	Traceback (most recent call last):
	  File "/opt/conda/bin/synthstrip", line 8, in <module>
	    sys.exit(main())
	  File "/opt/conda/lib/python3.9/site-packages/mriqc/synthstrip/cli.py", line 125, in main
	    conformed = conform(image)
	  File "/opt/conda/lib/python3.9/site-packages/mriqc/synthstrip/cli.py", line 224, in conform
	    nii = Affine(
	  File "/opt/conda/lib/python3.9/site-packages/nitransforms/base.py", line 276, in apply
	    data = np.asanyarray(
	  File "/opt/conda/lib/python3.9/site-packages/nibabel/arrayproxy.py", line 391, in __array__
	    arr = self._get_scaled(dtype=dtype, slicer=())
	  File "/opt/conda/lib/python3.9/site-packages/nibabel/arrayproxy.py", line 358, in _get_scaled
	    scaled = apply_read_scaling(self._get_unscaled(slicer=slicer), scl_slope, scl_inter)
	  File "/opt/conda/lib/python3.9/site-packages/nibabel/volumeutils.py", line 961, in apply_read_scaling
	    arr = arr + inter
	numpy.core._exceptions._ArrayMemoryError: Unable to allocate 164. MiB for an array with shape (224, 300, 320) and data type float64
Traceback:
	RuntimeError: subprocess exited with code 1.

